locales/en/public.json

```diff
@@ -37,7 +37,7 @@
     "STALE_REPORT": {
       "TEXT_one": "Most recent data from {{ count }} {{ units }} ago.",
       "TEXT_other": "Most recent data from {{ count }} {{ units }}s ago.",
-      "TOOLTIP": "Report data is stale. Click the Create Recording button and choose an option to start an active recording to source automated reports from."
+      "TOOLTIP": "Report data is stale. Click the Create recording button and choose an option to start an active recording to source automated reports from."
```

Comment:
I wonder if this needs some kind of styling applied so that the name/label of the button is more obvious? It could be quoted, or perhaps italicized?

Reply:
Yes that makes sense. But the button on the UI  is `Create` without the Recording . Another question that I had was for cases like `Active Recordings` should it be `
1. Active Recordings` or `
2. Active recordings`? 
I am leaning towards option 2 more.

Reply:
Seems similar to `Event Templates`/`Automated Rules` in the other comment thread. If anything, `Recording` is the custom resource in this case, and "active" is just an adjective describing them, as opposed to "archived" Recordings. But I think there's a reasonable case for considering Active Recording and Archived Recording as two different (related) custom resources which deserve capitalization.